src/Effects.js

```diff
@@ -743,6 +810,98 @@ module.exports = class Effects {
       }
     };
 
+    this.stars = {
+      star: [],
+      draw: function () {
+        p5.background('black');
+        let star = {
+          x: p5.random(0, 400),
+          y: p5.random(0, 400),
+          size: p5.random(15, 30),
+        };
+        this.star.push(star);
+        p5.noStroke();
+        p5.fill(255, 255, 0);
+        this.star.forEach(function (star){
+          p5.push();
+          p5.translate(star.x, star.y);
+          for (let i = 0; i < 3; i++) {
+            p5.rotate(360 / 5);
+            p5.ellipse(0, 0, 1, star.size);
+          }
+          let fadeSpeed = p5.map(star.size, 15, 30, 1, 2);
+          star.size = star.size - fadeSpeed;
+          star.y = star.y - 2;
+          p5.pop();
+        });
+        this.star = this.star.filter(function (star) {
+          return star.size > 0.1;
+        });
```

Comment:
If I'm reading this correctly, your shortest-lived stars last 73 frames, and your longest-lived stars last 83 frames, so we should expect to be drawing about 78 stars on any given frame.  Does that sound right?

I wonder if we should pass the optional `withinBounds` flag to [`p5.map`](https://p5js.org/reference/#/p5/map) so that the lowest fadeSpeed is 1.

Reply:
discussed improving in next version.